bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/entity/pcm/PCMActionSequence.java

```diff
@@ -147,20 +179,8 @@ public int compareTo(PCMActionSequence otherSequence) {
 		} else if (!this.getRequiredDatabases().isEmpty() && otherSequence.getRequiredDatabases().isEmpty()) {
 			return 1;
 		} else {
-			List<String> requriredDatabases = this.getRequiredDatabases();
-			List<String> providedDatabases = this.getProvidedDatabases();
-			
-			List<String> otherRequiredDatabases = otherSequence.getRequiredDatabases();
-			List<String> otherProvidedDatabases = otherSequence.getProvidedDatabases();
-			
-			if (requriredDatabases.containsAll(otherProvidedDatabases)) {
-				return 1;
-			} else if (otherRequiredDatabases.containsAll(providedDatabases)) {
-				return -1;
-			} else {
-				logger.error("Found incompatible set of Databases, Action Sequences depend on each other");
-				throw new IllegalStateException("Cylic loop of databases found in action sequences");
-			}
+			logger.error("Found incompatible set of Databases, Action Sequences depend on each other");
```

Comment:
The error message should also contain the information of the *cyclic* dependency